rosie/src/main/java/com/karumi/rosie/view/IRosieActivity.java

```diff
@@ -0,0 +1,5 @@
+package com.karumi.rosie.view;
```

Comment:
Can you also add the company copyright header we are using in the rest of the files?